### PR TITLE
US002 + US006 - Job Spec page

### PR DESCRIPTION
**Pull request by RyanCunning1**
US002 + US006 Creation of Job specification page with both java and UI testing, also includes information on responsibilities for each role

### CODE REVIEW

File: Views/Roles.html [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+            <br>
+            <br>
+
+            <h3 class="text-center-block__title">Competencies For Band Level</h3>
+            <p><a href="http://localhost:7999/jobslist/">Basic–Trainee</a></p>
+            <p><a href="http://localhost:7999/jobslist/">Foundational– Associate</a></p>
+            <p><a href="http://localhost:7999/jobslist/">Developing–Senior Associate</a></p>
+            <p><a href="http://localhost:7999/jobslist/">Intermediate–Consultant</a></p>
+
+
```
Comment (liammc98) [inline Views/Roles.html:421]: Remove whitespace here & above please

File: Views/UITests/src/main/java/pages/RolesPage.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    public String tableResponsibilityCheck(){
+        return tableResponsibility.getText();
+    }
+
+    public String tableJobSpecURLCheck(){
+        return tableJobSpecURL.getText();
+    }
+
+
```
Comment (liammc98) [inline Views/UITests/src/main/java/pages/RolesPage.java:43]: Should probably remove whitespace & the space below RolesPage() method

Comment (John-Neill-Kainos) [inline Views/UITests/src/main/java/pages/RolesPage.java:43]: Delete white space

File: application/apiService/src/main/java/com/kainos/ea/apiService/controllers/apiService.java [hunk 3]
```diff
@@ -382,6 +386,64 @@ public static JSONArray getJobFamilyUsingCapabilityID(String capabilityID){
         return jobFamilyJSON;
     }
 
+
+    public static JSONArray getRolesPageWithJobRoleName(String roleName){
```
Comment (liammc98) [inline application/apiService/src/main/java/com/kainos/ea/apiService/controllers/apiService.java:390]: Remove & test other getJobSpec() method

**Pull request closed without merging**